src/mwoffliner.lib.ts

```diff
@@ -211,7 +211,8 @@ async function execute(argv: any) {
 
   MediaWiki.apiCheckArticleId = mwMetaData.mainPage
   await MediaWiki.hasCoordinates(downloader)
-  await MediaWiki.hasWikimediaDesktopRestApi()
+  await MediaWiki.hasWikimediaDesktopApi()
+  const hasWikimediaMobileApi = await MediaWiki.hasWikimediaMobileApi()
```

Comment:
In the next few lines, we have twice `await downloader.setBaseUrls(forceRender)`, looks like to be useless. Once should be enough.